pages/en/billing.mdx

```diff
@@ -1,12 +1,12 @@
 ---
-title: Billing on Subgraph Studio
+title: Billing
 ---
 
 ## Billing on Arbitrum
 
 > Invoices are generated on a weekly basis.
 
-While The Graph protocol operates on Ethereum Mainnet, the billing contract lives on the [Arbitrum](https://arbitrum.io/) network to reduce transaction times and cost. You'll be required to pay fees based on the query fees you generate using your API keys. Using the billing contract, you'll be able to:
+While The Graph protocol operates on Ethereum Mainnet, [the billing contract](https://arbiscan.io/address/0x1b07d3344188908fb6deceac381f3ee63c48477a) lives on the [Arbitrum](https://arbitrum.io/) network to reduce transaction times and cost. You'll be required to pay fees based on the query fees generated when using your API keys. Using the billing contract, you'll be able to:
```

Comment:
"You'll be required to pay fees based on the query fees generated when using your API keys" sounds funny to me. What about "You'll be required to pay the query fees generated from your API keys"?

Reply:
This sounds weird to me as well. "fees based on the query fees" leaves room for misinterpretation. Consumers may think they need to pay surcharges etc. Additionally I'd like to point out that the first bulletpoint use the term "wallet", while the rest of the page use "account". Note that the studio itself uses "Billing balance". (And "Wallet balance")

Benoit's suggestion is good  "You'll be required to pay the query fees generated from your API keys"

Alternative paragraph: (feel free to ignore :) ):
Queries on the decentralized network is paid using GRT. Consumers can manage billing on the network in the Subgraph Studio. The Billing dashboard lets them add or remove GRT from a billing balance. It also gives them a full overview over transactions, invoices, and accrued query fees since the beginning of the current billing period. Invoices are generated weekly, and are paid automatically using GRT in the billing balance. 

  - While The Graph protocol operates on Ethereum Mainnet, [the billing contract](https://arbiscan.io/address/0x1b07d3344188908fb6deceac381f3ee63c48477a) lives on the [Arbitrum](https://arbitrum.io/) network to reduce transaction times and cost.

Reply:
Thank you both! Adjusted the copy to include Benoit's suggestion & changed the references of "account" to "billing balance." 

Going to reach out to Nena for any last minute feedback on this.

Reply:
Thank you 🙌 
The first bullet point still refer to the "billing balance" as the "wallet". This is is particularly confusing, since "wallet" and "wallet balance" is already used in the studio for the users ethereum account (And in web3, this is commonly referred to as the users "wallet")

I would once again recommend changing
- Add and remove GRT from your wallet.

to 
- Add and remove GRT from your billing balance.